packages/plugins/src/hardhat/tasks.ts

```diff
@@ -1059,9 +1044,14 @@ task(TASK_NODE)
 
 task(TASK_TEST)
   .addFlag('show', 'Show ChugSplash deployment information')
+  .addFlag('confirm', 'Automatically confirm contract upgrades')
```

Comment:
Imo, this flag should only be necessary if the user is on a live network. I'd personally find it annoying to include a `--confirm` flag whenever I'm using this command to test an upgrade on the Hardhat network. A solution to this would be:
1. Change this flag description to be something like: `Automatically confirm contract upgrades. Only applicable if upgrading on a live network.`
2. Inside this task, set `confirm` to `true` if we're on the Hardhat network.

In the future, we could use the CI deployment task instead of this solution, but I think this is sufficient for now.

Reply:
Since the test task only ever deploys contracts when run against the hardhat network, I ended up just removing the confirm flag like the node task.